### PR TITLE
Update Secureserver.net.xml

### PR DESCRIPTION
**Pull request by koops76**
#11114

### CODE REVIEW

File: src/chrome/content/rules/Secureserver.net.xml [hunk 1]
```diff
@@ -1,45 +1,16 @@
-
-<!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://hxcc.secureserver.net/ => https://hxcc.secureserver.net/: (28, 'Connection timed out after 20000 milliseconds')
```
Comment (J0WI) [inline src/chrome/content/rules/Secureserver.net.xml]: Please add a comment for the timeout

File: src/chrome/content/rules/Secureserver.net.xml [hunk 2]
```diff
@@ -1,45 +1,16 @@
...
-Fetch error: http://p3smysqladmin01.secureserver.net/ => https://p3smysqladmin01.secureserver.net/: (60, 'SSL certificate problem: unable to get local issuer certificate')
```
Comment (J0WI) [inline src/chrome/content/rules/Secureserver.net.xml]: Please add a comment for the incomplete cert chain.

**Comment by J0WI** [comment]
@koops76 I still miss the comments

**Comment by koops76** [comment]
@J0WI Done.

**Pull request merged**